main/Main.hs

```diff
@@ -94,10 +110,11 @@ fileTarget path = Target (readFileUtf8 path) path atomicWriteFile
 checkFileTarget :: FilePath -> Target
 checkFileTarget path = Target (readFileUtf8 path) path (const $ pure ())
 
-toTargets :: Nixfmt -> [Target]
-toTargets Nixfmt{ files = [] }    = [stdioTarget]
-toTargets Nixfmt{ check = False, files = paths } = map fileTarget paths
-toTargets Nixfmt{ check = True, files = paths } = map checkFileTarget paths
+toTargets :: Nixfmt -> IO [Target]
+toTargets Nixfmt{ files = [] }    = pure [stdioTarget]
+toTargets Nixfmt{ files = ["-"] } = pure [stdioTarget]
```

Comment:
Did you check if this case gets hit? I think there's a good chance the single dash will be interpreted as some kind of flag and not end op in the positional arguments.

Reply:
Yes.
Previous:
```nixfmt: -: openFile: does not exist (No such file or directory)```
New:
```
❯ dist-newstyle/build/x86_64-linux/ghc-8.10.7/nixfmt-0.5.0/x/nixfmt/build/nixfmt/nixfmt - <<< "hello"
hello
```